bat_vis1.Rmd

```diff
@@ -0,0 +1,55 @@
+---
+title: "Bat_vis1"
+author: "Heather S. Davies"
+date: "April 11, 2018"
+output: html_document
+---
+
+
+load the tidyverse library and ggrepel
+
+```{r libraries}
+library(dplyr)
+library(tidyverse)
+library(ggrepel)
+library(ggplot2)
+library(ggthemes)
+
+```
+
+load data
+
+```{r load_data}
+
+batdata <- read.csv("Bat_cty_vir.csv")
+
+View(batdata)
+
+```
+
+ 
+ 
+ ```{r summary-viz}
+batdata %>% 
+ 
+  group_by(taxonid) %>% 

```

Comment:
This looks like a great plot (I don't really know until I run it though)

Reply:
Thanks - it is a dummy file